packages/react-charting/src/components/HeatMapChart/__snapshots__/HeatMapChart.test.tsx.snap

```diff
@@ -1378,6 +1382,7 @@ exports[`HeatMapChart snapShot testing renders hideLegend correctly 1`] = `
           overflow: auto;
         }
     data-focuszone-id="FocusZone3"
+    data-tabster="{\\"uncontrolled\\": {}}"
```

Comment:
this should get resolved after merging master

Reply:
Some tests were skipped before for which the snapshots were not updated.